ostd/src/mm/page_table/node/entry.rs

```diff
@@ -86,18 +86,34 @@ impl<'a, 'rcu, C: PageTableConfig> Entry<'a, 'rcu, C> {
     ///
     /// The old child is returned.
     ///
+    /// # Safety
+    ///
+    /// The caller must ensure that:
+    ///  - if the new child is `PageTable`, the address must point to a valid
+    ///    page table node, the level must be correct and the pointed node must
+    ///    outlive this node;
+    ///  - if the new child is `Frame`, the new established mapping must be
+    ///    a valid mapping (TODO: specify valid mappings).
```

Comment:
Since this is a safe method, it cannot have safety requirements.

In fact, the safety conditions do not make sense. The first condition is already checked in the method body. The second condition is not even part of the safety contract of the APIs under `page_table/node`.